docs/src/main/asciidoc/logging.adoc

```diff
@@ -154,62 +203,84 @@ WARNING:: Same as **WARN**.
 CONFIG:: Service configuration information.
 FINE:: Same as **DEBUG**.
 FINER:: Same as **TRACE**.
-FINEST:: Event more debugging information than `TRACE`, maybe with even higher frequency.
+FINEST:: Even more debugging information than `TRACE`, with maybe even higher frequency.
+
 
 == Runtime configuration
 
-Run time logging is configured in the `application.properties` file,
-for example, to set the default log level to `INFO` logging and include Hibernate `DEBUG` logs:
+Runtime logging is configured in the `application.properties` file.
 
+Quarkus internally employs `jboss-logging` and provides link:https://quarkus.io/developer-joy/[unified configuration] for all <<supported-logging-apis,supported logging APIs>>.
+
+.An example of setting the default log level to `INFO` logging and include Hibernate `DEBUG` logs:
 [source, properties]
 ----
 quarkus.log.level=INFO
 quarkus.log.category."org.hibernate".level=DEBUG
 ----
 
-Setting a log level below `DEBUG` requires the minimum log level to be adjusted,
-either globally via the `quarkus.log.min-level` property or per-category as shown in the example above,
-as well as adjusting the log level itself.
+Setting a log level below `DEBUG` requires the minimum log level to be adjusted either:
+
+* per category, as shown in the previous example
+* globally, by using the `quarkus.log.min-level` property
+* directly, by adjusting the log level directly
 
-Minimum logging level sets a floor level that Quarkus will be needed to potentially generate,
-opening the door to optimization opportunities.
-As an example, in native execution the minimum level enables lower level checks (e.g. `isTraceEnabled`) to be folded to `false`,
-resulting in dead code elimination for code that will never to be executed.
+The minimum logging level sets a floor level that Quarkus needs to generate, potentially opening the door to optimization opportunities.
+
+.An example from native execution:
+Setting the `INFO` as the minimum logging level sets lower-level checks, such as `isTraceEnabled`, to `false`.
```

Comment:
```suggestion
Setting `INFO` as the minimum logging level sets lower-level checks, such as `isTraceEnabled`, to `false`.
```

Reply:
I would use even stronger language: "... causes check methods for lower log levels, such as `isTraceEnabled`, to always return a constant `false` value."